### PR TITLE
[chore] fix exhaustive lint for processor spanprocessor (#23266)

### PR DESCRIPTION
**Pull request by jaydeluca**
**Description:** <Describe what has changed.>
Related #23266

### CODE REVIEW

**Comment by jaydeluca** [comment]
i'm not sure why it's not skipping the changelog step, i have added `[chore]` to the title but I am unable to add the label

**Review by atoulme — APPROVED** [review]

**Comment by fatsheep9146** [comment]
ping owner for approve @boostchicken

File: processor/spanprocessor/span.go [hunk 1]
```diff
@@ -142,6 +142,10 @@ func (sp *spanProcessor) processFromAttributes(span ptrace.Span) {
 			sb.WriteString(strconv.FormatFloat(attr.Double(), 'f', -1, 64))
 		case pcommon.ValueTypeInt:
 			sb.WriteString(strconv.FormatInt(attr.Int(), 10))
+		case pcommon.ValueTypeEmpty:
+		case pcommon.ValueTypeSlice:
+		case pcommon.ValueTypeBytes:
+		case pcommon.ValueTypeMap:
```
Comment (codeboten) [inline processor/spanprocessor/span.go:148]: These should all have `fallthrough`

**Review by codeboten — CHANGES_REQUESTED** [review]

File: processor/spanprocessor/span.go (hunk 1 above)
Comment (atoulme) [reply processor/spanprocessor/span.go:148]: woops my Java is showing. Sorry I missed this.

**Review by codeboten — APPROVED** [review]
Thanks @jaydeluca!

**Review by codeboten — APPROVED** [review]

**Pull request merged**